resources/views/pages/home-participant.blade.php

```diff
@@ -1,23 +1,80 @@
 @extends('master')
 
+@php 
+	use App\Helpers\DateHelper; 
+	use Carbon\Carbon;
+@endphp
+
 @section('title')
 	Home
 @endsection
 
 @section('content')
 
 <div class="jumbotron">
-	<h1>Welkom, {{ Auth::user()->profile->voornaam }}!</h1>
+	<h1>Welkom, {{ $participant->voornaam }}!</h1>
+
 	<p>
-		Leuk je weer te zien. Ga je weer een keer mee op kamp?
+		Leuk je weer te zien. 
+		@if ($participant->geboortedatum->isBirthday())
+			<b>Gefeliciteerd met je verjaardag!</b> <span class="glyphicon glyphicon-gift"></span>
+		@endif
 	</p>
-	@if ($congrats == 1)
+
+	@forelse ($registeredCamps as $c)
+	<div class="panel panel-default">
+		<div class="panel-body">
+			<h2>
+				Status inschrijving <a href="{{ url('events', $c->id) }}">{{ $c->naam }}</a> ({{ DateHelper::Format($c->datum_start) }} t/m {{ DateHelper::Format($c->datum_eind) }}):
+			</h2>
+			<hr/>
+			<p>
+				@if ($c->pivot->isPaid())
+					<span class="glyphicon glyphicon-ok" aria-hidden="true"></span> &nbsp;
+					Betaling ontvangen op {{ DateHelper::Format(new Carbon($c->pivot->datum_betaling)) }}.
+				@else
+					<span class="glyphicon glyphicon-warning-sign" aria-hidden="true"></span> &nbsp;
+					Betaling nog niet ontvangen! <a href="{{ url('pay', $c->id) }}">Betaal direct @money($c->pivot->getPayment()->getTotalAmount()) via iDeal.</a>
+				@endif
+			</p>
+			<p>
+				@if ($participant->inkomen == 0)
```

Comment:
note: hoe dit leest: "als je geen inkomen heb, krijg je geen korting" . XD

Reply:
Laten we het erop houden dat de Robin in 2015 nog niet echt nadacht over zaken als naamgeving. Of uberhaupt clean code. Het werkte :')

Reply:
ik vond het gewoon mooi :P